### ISSUE TITLE
Outdated dependency to @sanity/ui

### ISSUE THREAD

**Issue by andersboe** [sanity-io/document-internationalization#46]
Hi,

We're having some dependency trouble building our studio with `npm install`.

`@sanity/default-layout` is currently depending on `^0.37.x` of `@sanity/ui`, while `@sanity/document-internationalization"` is at `^0.36.12`

Could this be bumped? Thanks in advance! 

(using `node v17.8.0` and `npm 8.5.5`)
```
  npm ERR! Found: @sanity/ui@0.37.8
  npm ERR! node_modules/@sanity/ui
  npm ERR!   @sanity/ui@"^0.37.6" from @sanity/base@2.29.5
  npm ERR!   node_modules/@sanity/base
  npm ERR!     @sanity/base@"2.29.5" from the root project
  npm ERR!     @sanity/base@"2.29.5" from @sanity/default-layout@2.29.5
  npm ERR!     node_modules/@sanity/default-layout
  npm ERR!       @sanity/default-layout@"2.29.5" from the root project
  npm ERR!     7 more (@sanity/default-login, @sanity/desk-tool, ...)
  npm ERR!   @sanity/ui@"^0.37.6" from @sanity/default-layout@2.29.5
  npm ERR!   node_modules/@sanity/default-layout
  npm ERR!     @sanity/default-layout@"2.29.5" from the root project
  npm ERR!   5 more (@sanity/default-login, @sanity/desk-tool, ...)
  npm ERR! 
  npm ERR! Could not resolve dependency:
  npm ERR! @sanity/document-internationalization@"0.1.7" from the root project
  npm ERR! 
  npm ERR! Conflicting peer dependency: @sanity/ui@0.36.17
  npm ERR! node_modules/@sanity/ui
  npm ERR!   peer @sanity/ui@"^0.36.12" from @sanity/document-internationalization@0.1.7
  npm ERR!   node_modules/@sanity/document-internationalization
  npm ERR!     @sanity/document-internationalization@"0.1.7" from the root project
```

**Comment by snorrees** [comment]
Thanks for the feedback, we are bumping sanity/ui in this PR: https://github.com/sanity-io/document-internationalization/pull/47 which will go out sometime tomorrow.

**Comment by snorrees** [comment]
Should be fixed in [0.3.0](https://github.com/sanity-io/document-internationalization/releases/tag/0.3.0)
